src/zzz_od/operation/challenge_mission/check_next_after_battle.py

```
@@ -17,18 +18,34 @@ def __init__(self, ctx: ZContext, try_next: bool):
-    @operation_node(name='无再来一次')
-    def finish(self) -> OperationRoundResult:
         return self.round_by_find_and_click_area(self.last_screenshot, '战斗画面', '战斗结果-完成',
                                                  success_wait=5, retry_wait=1)
 
+    @node_from(from_name='判断再来一次', status='战斗结果-再来一次')
+    @operation_node(name='恢复电量')
+    def restore_charge(self) -> OperationRoundResult:
+        # 检查是否在恢复电量界面
+        result = self.round_by_find_area(self.last_screenshot, '实战模拟室', '恢复电量')
+        if not result.is_success:
+            # 没有弹窗，直接返回再来一次的结果
```

Ruff (0.13.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)